main.py

```diff
@@ -1,7 +1,41 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        "Western",
+        "Action",
+        "Dramma",
+    ]
+
+    actors = [
+        ("George", "Klooney"),
```

Comment:
There is a typo in the actor's name "Klooney". It should be "Clooney". This is corrected later in the code, but it's better to avoid such typos initially.